doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -456,12 +498,59 @@ Password-authenticated key agreement driver configurations
 
 You can use the following PSA driver-specific configurations for fine-grained control over which drivers provide password-authenticated key agreement support.
 
-+-----------------------+--------------------------+---------------------------------------------------------------+
-| PAKE algorithm        | nrf_cc3xx driver support | nrf_oberon driver support                                     |
-+=======================+==========================+===============================================================+
-| EC J-PAKE             | Not supported            | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_JPAKE_OBERON`   |
-+-----------------------+--------------------------+---------------------------------------------------------------+
-| SPAKE2+               | Not supported            | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SPAKE2P_OBERON` |
-+-----------------------+--------------------------+---------------------------------------------------------------+
-| SRP                   | Not supported            | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SRP_OBERON`     |
-+-----------------------+--------------------------+---------------------------------------------------------------+
++-----------------------+--------------------------+---------------------------+
+| PAKE algorithm        | nrf_cc3xx driver support | nrf_oberon driver support |
++=======================+==========================+===========================+
+| EC J-PAKE             | Not supported            | Supported                 |
++-----------------------+--------------------------+---------------------------+
+| SPAKE2+               | Not supported            | Supported                 |
++-----------------------+--------------------------+---------------------------+
+| SRP                   | Not supported            | Supported                 |
++-----------------------+--------------------------+---------------------------+
+
+The driver configuration of the :ref:`nrf_security_drivers_oberon` driver is automatically generated based on the user enabled algorithms in  `Password-authenticated key agreement configurations`_.
+
+Key size configurations
+***********************
+
+We support key size configuration options for AES and RSA keys.
```

Comment:
Refer to the product or module instead of using 'we'.

Oberson PSA supports?

Reply:
I will write nrf_security, I think that it fits better since it includes the Oberon PSA core.